demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import Message from '@gdbots/pbj/Message';
+import { STATUS_FULFILLED, STATUS_PENDING } from '@triniti/app/constants';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  StatusMessage,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
+      accessToken: PropTypes.string.isRequired,
+      collaborationNodes: PropTypes.arrayOf(PropTypes.instanceOf(Message)).isRequired,
+      contentType: PropTypes.oneOf(['articles', null]),
+      handleRequestCollaborationNodes: PropTypes.func.isRequired,
+      title: PropTypes.string.isRequired,
+    };
+
+    static defaultProps = {
+      contentType: null,
+    };
+
+    constructor(props) {
+      super(props);
+      this.handleRequestCollaborationNodes = this.handleRequestCollaborationNodes.bind(this);
+      this.state = { status: STATUS_FULFILLED };
+    }
+
+    componentDidMount() {
+      this.handleRequestCollaborationNodes();
+    }
+
+    async handleRequestCollaborationNodes() {
+      const { handleRequestCollaborationNodes, accessToken } = this.props;
+      this.setState({ status: STATUS_PENDING });
+      await handleRequestCollaborationNodes(accessToken);
+      this.setState({ status: STATUS_FULFILLED });
+    }
+
+    render() {
+      const { collaborationNodes, contentType, title } = this.props;
+      const { status } = this.state;
+      const articleNodes = collaborationNodes.filter((node) => node.schema().getCurie().getMessage() === 'article').map((node) => (node));
+      const otherNodes = collaborationNodes.filter((node) => node.schema().getCurie().getMessage() !== 'article').map((node) => (node));
```

Comment:
@nanetteCodes i dont think the `.map((node) => (node));` is necessary here. also you can use `.reduce` to do this all in one call, something like

```
const { articleNodes, otherNodes } = collaborationNodes.reduce((acc, cur) => {
// sort em out
}. { articleNodes: [], otherNodes: [] })
```